cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/validator/AwsIDBrokerObjectStorageValidator.java

```diff
@@ -58,30 +63,32 @@ public ValidationResult validateObjectStorage(AmazonIdentityManagementClient iam
             if (instanceProfile != null) {
                 CloudIdentityType cloudIdentityType = cloudFileSystem.getCloudIdentityType();
                 if (CloudIdentityType.ID_BROKER.equals(cloudIdentityType)) {
-                    validateIDBroker(iam, instanceProfile, cloudFileSystem, resultBuilder);
+                    validateIDBroker(iam, instanceProfile, cloudFileSystem, backupLocationBase, resultBuilder);
```

Comment:
Do we need to validate the backup location for both id_broker and log? It means nothing but on the UI the backup location base settings is grouped together with the log location settings.

Reply:
Yes, 
DATALAKE_ADMIN_ROLE, RANGER_AUDIT_ROLE, and LOG_ROLE need permissions on the backup location.

Reply:
ok